app/models/video.rb

```diff
@@ -7,5 +7,4 @@ def self.search_by_title(title)
     Video.where('title LIKE ?', "%#{title}%").order(created_at: :desc)
```

Comment:
thanks, I was wondering about this.  Everything in my db is saved in lowercase, so not a problem, but that wouldn't account for when other people try to use this code.  I'll change this.